### PR TITLE
Ask BL users to migrate to GCB

### PR DESCRIPTION
**Pull request by kienstra**
#### Changes
* This notice will supersede the previous notice.
* The previous notice will never display.
* But even if they've dismissed the previous notice, this will show.
* Still, this will only show on specific pages, not on every `/wp-admin` page.

#### Testing instructions
1. `composer install && npm install && npm run dev`
2. Go to `/wp-admin` > Block Lab
3. Expected: the new notice appears here-are-content-blocks
4. Click 'Migrate'
5. Expected: the new migration copy and `.gif` appear:
Screen Shot 2021-03-25 at 9 57 37 AM

### CODE REVIEW

**Comment by kienstra** [comment]
It looks like Travis is out, I think there's a limit on free plans now. 

It might not be worth it to set up GitHub actions.

For now, `npm run lint && npm run test:js` should be alright.

**Comment by lukecarbis** [comment]
@kienstra What's the reason for kicking off this PR?

**Comment by kienstra** [comment]
Hi @lukecarbis,
Ah, I should have linked to [GF-2789](https://wpengine.atlassian.net/browse/GF-2789). 

The idea is to let Block Lab users know that GCB is now much better than BL, with the new editor. Ideally, this could convince some of the 9,000 users to migrate.

As you know, when we first released GCB, it wasn't really better than GCB.

**Comment by kienstra** [comment]
Here's the new migration notice copy:

here-are-content-blocks

Here's the new migration page copy:
Screen Shot 2021-03-25 at 9 57 37 AM

**Comment by kienstra** [comment]
Todo: 

- [x] Add the new wp.org banner image
- [x] Add the `.gif` to the migration page

**Comment by kienstra** [comment]
Hi @RobStino and @lukecarbis,
Could you please review this? 

Travis isn't running anymore, I think you have to pay to use it at the rate we did.

So feel free to run linting or testing on your own if you'd like: `npm run lint && npm run test:js`

**Review by RobStino — APPROVED** [review]

**Review by dreamwhisper — APPROVED** [review]
Looks good!
Screen Shot 2021-03-26 at 2 04 35 PM
Screen Shot 2021-03-26 at 2 04 46 PM

**Comment by kienstra** [comment]
@dreamwhisper,
Thanks so much!

File: js/migration/components/intro.js [hunk 1]
```diff
@@ -16,22 +17,24 @@ import { __ } from '@wordpress/i18n';
...
-				<p>{ __( 'To continue receiving the best of what our team is building, we encourage you to migrate over. Our migration tool makes this nice and easy, and for the majority of use cases, completely automated.', 'block-lab' ) }</p>
+				<p>{ __( 'We’ve been busy! Since the Block Lab team joined WP Engine in 2020, the new plugin (Genesis Custom Blocks) has got a whole lot better. Plenty of things under the hood, but the biggest change is the new Block Builder interface.', 'block-lab' ) }</p>
+				<p>{ __( 'Check it out below!', 'block-lab' ) }</p>
+				<img
+					width={ 800 }
+					height={ 534 }
+					src={ editorGifUrl }
+					alt={ __( 'New editor screencast', 'block-lab' ) }
+				/>
+				<p>{ __( 'Migrating from Block Lab to Genesis Custom Blocks is super easy, seamless, and (for 95% of users) completely automated.', 'block-lab' ) }</p>
```
Comment (johnstonphilip) [inline js/migration/components/intro.js]: Just curious about this 95% number. Is that accurate?

Comment (kienstra) [reply js/migration/components/intro.js]: Good question. The 5% would probably be either people that have used PHP filters or where there is a bug in migration. The PHP filters aren't that common.

Comment (kienstra) [reply js/migration/components/intro.js]: Do you think 'the vast majority' would convey this better?

**Comment by kienstra** [comment]
I disabled Travis. It wasn't running anyway, I think this repo went over the free runs they allow.

**Pull request merged**